icons/swiprolog.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>SWI Prolog</title><path d="M4.584 0c-.28 0-.568.025-.865.076.168.258.337.489.492.723.455.684.489 2.2.123 3.28a10.63 10.63 0 0 0-3.02 7.437c0 1.359.26 2.66.723 3.86.117.329.22.66.332.989.099.305.192.607.281.906.258.885.418 1.803.442 2.726.01.253.003.507-.006.76-.005.09-.003.172-.008.262l2.164-1.983c.394.361.928.688 1.57.965a2.513 2.513 0 0 0-1.078 2.066c0 .731.315 1.39.807 1.848a1.755 1.755 0 0 1 1.678-2.26A1.753 1.753 0 0 1 9.867 24c.553-.464.9-1.158.9-1.932 0-.426-.108-.824-.29-1.175a17.93 17.93 0 0 0 3.047.053 2.51 2.51 0 0 0 .538 2.97 1.755 1.755 0 0 1 1.678-2.26A1.753 1.753 0 0 1 17.39 24c.553-.464.9-1.154.9-1.932 0-.773-.346-1.46-.894-1.925.665-.253 1.232-.557 1.668-.899l1.868 1.71c-.005-.09-.005-.174-.01-.263a20.471 20.471 0 0 1-.004-.76c.028-.923.182-1.836.44-2.725.088-.3.182-.6.28-.904.113-.347.226-.698.348-1.045.46-1.162.7-2.424.7-3.74 0-2.915-1.172-5.56-3.06-7.485-.352-1.073-.31-2.557.134-3.232.154-.234.324-.465.502-.723-1.583-.272-2.93.183-3.833 1.313-.108.136-.21.22-.32.257a10.61 10.61 0 0 0-4.104-.82c-1.462 0-2.854.295-4.12.824-.112-.037-.22-.12-.331-.261C6.822.47 5.793-.001 4.584 0ZM17 4.008A4.23 4.23 0 0 1 21.23 8.24a4.23 4.23 0 0 1-6.954 3.239c-.09.42-.304.778-.515 1.18-.29.549-.577 1.097-.867 1.65l-.862 1.587c-.21-.313-.39-.668-.553-.992-.187-.365-.38-.736-.572-1.101-.365-.7-.91-1.411-1.092-2.203a4.23 4.23 0 1 1-.924-7.14c.24.102.47.223.69.368A4.21 4.21 0 0 1 11.4 8.29c0 .206-.014.409-.046.605a2.296 2.296 0 0 1 1.467.037 4.12 4.12 0 0 1-.057-.683c0-.366.052-.723.136-1.06l.014-.051c.023-.085.052-.168.08-.252l.06-.174.028-.07A4.227 4.227 0 0 1 17 4.008ZM7.975 6.512c-1.177 0-2.127.95-2.127 2.127 0 1.176.95 2.127 2.127 2.127a2.125 2.125 0 0 0 2.127-2.127 2.125 2.125 0 0 0-2.127-2.127zm9.869.084a2.128 2.128 0 1 0 .002 4.256 2.128 2.128 0 0 0-.002-4.256zm-9.82.81c.714 0 1.287.577 1.287 1.29a1.28 1.28 0 0 1-1.286 1.283 1.286 1.286 0 1 1 0-2.573zm9.746.018c.712 0 1.283.577 1.283 1.289s-.576 1.285-1.283 1.285a1.288 1.288 0 1 1 0-2.574zm-10.325.445a.362.362 0 0 0-.361.36c0 .198.164.363.36.363.198 0 .363-.16.363-.362a.365.365 0 0 0-.362-.36Zm9.739.006a.362.362 0 0 0-.362.36c0 .198.165.36.362.36a.356.356 0 0 0 .359-.36.36.36 0 0 0-.36-.36zm-5.157 1.344a1.98 1.98 0 0 0-1.982 1.982c0 .877.62 1.63 1.004 2.37.164.32.332.64.496.958.145.277.3.585.482.862l.746-1.377c.249-.478.502-.956.75-1.434.244-.454.487-.85.487-1.383a1.98 1.98 0 0 0-1.983-1.978Zm-9.869 4.283.53.488.826-.29-.957 1.003Zm19.737.146-.396 1.2-.957-1.002.824.289.53-.487zm-17.825.127.492.53.844-.23-1.031.927Zm15.915.145-.31 1.228-1.03-.927.842.228.498-.53zm-14.083.22.493.53.843-.23-1.031.927-.305-1.226Zm12.246.145-.308 1.229-1.027-.928.843.228zM2.764 15.56l.529.486.824-.29-.955 1.003-.398-1.2zm18.524.144-.398 1.2-.957-1.003.824.29zm-16.692.238.529.489.824-.291-.951 1.004Zm14.86.14-.4 1.202-.954-1.004.824.29zM3.271 17.65l.532.486.824-.29-.957 1.003zm17.503.145-.398 1.2-.955-1.004.824.291.53-.487z"/></svg>
```

Comment:
It doesn't match with the original. [See here!](https://github.com/simple-icons/simple-icons/assets/35000807/10fb07cc-0c5d-473e-9bc4-bbce4166881d)